book/website/reproducible-research/open/open-hardware.md

```diff
@@ -1,111 +1,359 @@
+
+- **Instructions for operation and maintenance**. Describe how hardware users should operate the hardware (for example how to calibrate and test it). Indicate any maintenance that should be done to secure a good functionality of the hardware.
+
+- **Repair and disposal**. Indicate where or how the hardware can be repaired, and indicate how to dispose or recycle the hardware, if it is beyond repair.
+
+> Note that producing documentation-quality pictures consistently requires adequate tools and setup.
+
+
+## Community documentation
+
+
+While open hardware communities are sometimes different than open oftware communities, the documentation useful to grow hardware communities are similar as the one for other open source project, see the  {ref}`pd-overview-repro` chapter. You may want to refer to the {ref}`collaboration<cl>`  and {ref}`cl-new-community`  guide of the turing way book for a more detailed description of certain aspects such as practices, metrics, behaviors and observables that can be related to thriving communities.
+
+OSH communities comes in all sizes and forms.
+They often develop around people facing similar needs, who realise they will get "something that none of [them] could have developed alone." (Interview: White rabbit, by the Open make team, Javier Serrano and Amanda Diez Frenandez.)
+We should therefore consider that developing good quality hardware and making it open source already entails an important aspect of community building.
+
+In the following section, we share some consideration about community building for OSH project.
+
+
+
+### Considerations for OSH community building
+
+While scoping your project, it is well advised to think about the different people who may engage with your OSH (see {ref}`cl-stakeholder-mapping`)
+Different OSH projects have included different partners at varying stages of their developement.
+On top of user and contributor roles that OSH have in common with open source software, local or global hardware manufacturers may become important partners of your project.
+You may also think early about the people who will eventually have to maintain and repair the hardware.
+To make it easier for them, it also helps to make your hardware designs modular (splitting your hardware in modules which may have alternative designs).
+Anoter specificity of hardware may be the importance of the creation of replication tutorials, workshops, seminars, or training material, which can impact the adoption of hardware designs.
+This is particularly relevant if the OSH is meant to be produced in Do-It-Yourself environments or as a teaching opportunity.
+
+```{note}
+Engineers is still a world full of tech-bros, who got irritated when one does not understand their jargon, and are not very welcoming with beginners and women. 
```

Comment:
```suggestion
Engineering culture can still be a closed one, not very welcoming to newcomers from different backgrounds.
```

Reply:
Fully agree with the sentiment, but not sure I'd use the exact words originally in the text for the book!